api/model/api.go

```diff
@@ -318,14 +318,21 @@ type MethodTransformMeta struct {
 	ToMethod HttpMethod `json:"to_method"`
 }
 
-type ValidatePathMeta struct {
-	// Allows override of default 422 Unprocessible Entity response code for validation errors.
-	ErrorResponseCode int        `json:"error_response_code"`
-	Path              string     `json:"path"`
-	Method            HttpMethod `json:"method"`
-	SchemaB64         string     `json:"schema_b64,omitempty"`
+// JSONValidationSchema represents 'schema' field of the 'validate_json'.
+// Validate JSON verifies user requests against a specified JSON schema and
+// check that the data sent to your API by a consumer is in the right format.
+type JSONValidationSchema struct {
```

Comment:
Instead of defining new type here, we can reuse the `ConfigDataType`. Just need to rename it to something generic.

Reply:
Okay @komalsukhani 👍 , what about `UnstructuredField`? Do you have any suggestions on your mind regarding the naming?

Reply:
@buraksekili It could be `MapStringInterfaceType`. Since we are using it specifically for that type.

Reply:
makes sense 👍 , i'll update the PR soon